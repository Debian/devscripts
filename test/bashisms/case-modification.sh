#!/bin/sh

foo=foo
bar=BAR

echo BASHISM: ${foo^f}
echo BASHISM: ${foo^^o}
echo BASHISM: ${bar,B}
echo BASHISM: ${bar,,R}

echo BASHISM: ${foo^}
echo BASHISM: ${foo^^}
echo BASHISM: ${bar,}
echo BASHISM: ${bar,,}

echo BASHISM: ${@^}
echo BASHISM: ${*^^}
echo BASHISM: ${@,}
echo BASHISM: ${*,,}
