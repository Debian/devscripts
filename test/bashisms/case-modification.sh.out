possible bashism in bashisms/case-modification.sh line 6 (${parm,[,][pat]} or ${parm^[^][pat]}):
echo BASHISM: ${foo^f}
possible bashism in bashisms/case-modification.sh line 7 (${parm,[,][pat]} or ${parm^[^][pat]}):
echo BASHISM: ${foo^^o}
possible bashism in bashisms/case-modification.sh line 8 (${parm,[,][pat]} or ${parm^[^][pat]}):
echo BASHISM: ${bar,B}
possible bashism in bashisms/case-modification.sh line 9 (${parm,[,][pat]} or ${parm^[^][pat]}):
echo BASHISM: ${bar,,R}
possible bashism in bashisms/case-modification.sh line 11 (${parm,[,][pat]} or ${parm^[^][pat]}):
echo BASHISM: ${foo^}
possible bashism in bashisms/case-modification.sh line 12 (${parm,[,][pat]} or ${parm^[^][pat]}):
echo BASHISM: ${foo^^}
possible bashism in bashisms/case-modification.sh line 13 (${parm,[,][pat]} or ${parm^[^][pat]}):
echo BASHISM: ${bar,}
possible bashism in bashisms/case-modification.sh line 14 (${parm,[,][pat]} or ${parm^[^][pat]}):
echo BASHISM: ${bar,,}
possible bashism in bashisms/case-modification.sh line 16 (${parm,[,][pat]} or ${parm^[^][pat]}):
echo BASHISM: ${@^}
possible bashism in bashisms/case-modification.sh line 17 (${parm,[,][pat]} or ${parm^[^][pat]}):
echo BASHISM: ${*^^}
possible bashism in bashisms/case-modification.sh line 18 (${parm,[,][pat]} or ${parm^[^][pat]}):
echo BASHISM: ${@,}
possible bashism in bashisms/case-modification.sh line 19 (${parm,[,][pat]} or ${parm^[^][pat]}):
echo BASHISM: ${*,,}
